_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,125 @@
+software.
+
+## News
+
+- **Payjoin adoption:** Chris Belcher [posted][belcher payjoin] to the
+  Bitcoin-Dev mailing list a request for people to look for ways to
+  increase [payjoin][topic payjoin] adoption along with a [wiki
+  page][payjoin wiki] tracking the projects that provide either sending
+  or receiving support for payjoin.  One [suggestion][raw payjoin] by
+  Craig Raw was to extend the protocol to allow it to work even when the
+  receiver doesn't operate a server.
+
+- **Making hardware wallets compatible with more advanced Bitcoin features:**
+  Kevin Loaec [started a discussion][loaec hww] on the Bitcoin-Dev
+  mailing list about how hardware wallets could be changed to allow them
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- [Bitcoin Core 0.21.0][Bitcoin Core 0.21.0] is a the next major version
+  of this full node implementation and its associated wallet and other
+  software.  Major new features include support for Tor version 3 hidden
+  services using [version 2 `addr` messages][topic addr v2], the optional
+  ability to serve [compact block filters][topic compact block filters],
+  support for [signets][topic signet] (including the default signet
+  which has [taproot][topic taproot] activated).  It also offers
+  experimental support for wallets that natively use [output script
+  descriptors][topic descriptors].  For a complete list of changes, see
+  the [release notes][bcc 0.21.0 notes].
+
+- [Rust Bitcoin 0.26.0][] is a new release of this library.  Major new
+  features include support for signet, version 2 `addr` messages,
+  and several improvements to [PSBT][topic psbt] handling.  See its
+  [release notes][rb notes] for details.
+
+- [BTCPay Server 1.0.6.7][] in the second maintenance release to
+  [1.0.6.5][btcpay server 1.0.6.5] from last week, which added "support
+  [for] a subset of output descriptors in the wallet setup" (see the
+  *notable changes* section below).  Other features and bug fixes were
+  also included.
+
+- [C-Lightning 0.9.3rc2][c-lightning 0.9.3] is a release candidate for a
+  new minor version of this LN node.
+
+- [LND 0.12.0-beta.rc5][LND 0.12.0-beta] is the latest release candidate
+  for the next major version of this LN node.
```

Comment:
maybe s/node/implementation/ here and line 74

Reply:
Why the longer word?

Reply:
Seems like the right word, e.g. a bitcoin node and a bitcoin implementation are different things to me. Do set me straight if they are equivalent, though.

Reply:
To me, a node is an implementation but an implementation is not necessarily a node, so "node" is the more specific term and should be preferred both for its brevity and specificity.

Reply:
I guess I see a node as an individual running instance of the software that is an implementation. So implementation is the term I would use to describe the software. That seems to be the usage I recall seeing in PR descriptions (example https://github.com/bitcoin/bitcoin/pull/20564) and in lightning-dev irc meeting discussions.

Reply:
Sure, and I also use "node" the same way as Pieter in that PR, but I'd also say that an HTML server sends pages to a web browser and also call Firefox a web browser, not a web implementation.  Does that make sense?

Reply:
Sounds reasonable to me.